modules/features/nova/src/main/kotlin/au/com/shiftyjelly/pocketcasts/nova/CatalogFactory.kt

```diff
@@ -30,13 +31,36 @@ internal class CatalogFactory(
         },
     )
 
+    fun newEpisodes(data: List<NovaLauncherNewEpisode>) = Catalog(
+        id = "NewReleases",
+        label = context.getString(LR.string.nova_launcher_new_releases),
+        items = data.map { episode ->
+            CatalogItem.Base(
+                id = episode.id,
+                intent = context.launcherIntent,
```

Comment:
This intent is to be clarified. Right now I used launching the app as a default.